scripts/bitbucket.py

```diff
@@ -91,11 +91,11 @@ def post_result(url, user, password, verify, data, debug):
         err("Commit: " + str(commit_sha))
 
     key = commit_sha
-    if j['params']['key']:
+    if j['params'].get('key', False):
```

Comment:
This is the only change that sticks out to me - I'm trying to read my mind 4 months in the past, but I think the idea here is that if the key doesn't exist (means we got JSON that didn't have it for some reason), the job will bomb right here with a KeyError and give the user appropriate feedback to look at their Concourse config.